src/javascript/app_2/Modules/Statement/Containers/statement.jsx

```diff
@@ -0,0 +1,65 @@
+import React                          from 'react';
+import { PropTypes as MobxPropTypes } from 'mobx-react';
+import PropTypes                      from 'prop-types';
+import Filter                         from './statement_filter.jsx';
+import NoActivityMessage              from '../Components/no_activity_message.jsx';
+import ListLargeScreen                from '../Components/list_large_screen.jsx';
+import ListSmallScreen                 from '../Components/list_small_screen.jsx';
+import { connect }                    from '../../../Stores/connect';
+import Loading                        from '../../../../../templates/_common/components/loading.jsx';
+
+class Statement extends React.Component {
+    componentDidMount()    { this.props.onMount(); }
+    componentWillUnmount() { this.props.onUnmount(); }
+    
+    render() {
+        const {
+            has_no_activity_message,
+            has_selected_date,
+            data,
+            is_loading,
+            is_mobile,
+        } = this.props;
+
+        return (
+            <React.Fragment>
+                <Filter />
+                { !is_mobile ?
```

Comment:
can we reverse the condition so we don't deal with `!`?

Reply:
@negarn 
Sure, thanks.